packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -1077,11 +1100,20 @@ function renderRoot(
           prepareFreshStack(root, expirationTime);
           return renderRoot.bind(null, root, expirationTime);
         }
-        const lastPendingTime = root.lastPendingTime;
-        if (lastPendingTime < expirationTime) {
+
+        const nextAfterSuspendedTime = root.nextAfterSuspendedTime;
```

Comment:
```suggestion
        const { nextAfterSuspendedTime } = root
```

Reply:
We also tend to avoid modern syntax as we don’t trust the transforms to produce good results.